<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addNewStudentButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewCourseButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Course</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="mDateTimeLabel">
       <property name="text">
        <string>Date : Time</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_courseTimetable">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Timetable</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCourseButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete selected course</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studentListButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View All Students</string>
    </property>
   </widget>
   <widget class="QTableView" name="studentTableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelUpcomingCourse">
       <property name="text">
        <string>Upcoming Course : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUpcomingCourseName">
       <property name="text">
        <string>CourseName</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAdministrator_mode"/>
    <addaction name="actionSwitch_to_student_mode"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdministrator_mode">
   <property name="text">
    <string>Switch to administrator mode</string>
   </property>
  </action>
  <action name="actionSwitch_to_student_mode">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Switch to student mode</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
